Makefile

```diff
@@ -173,6 +182,25 @@ create-aks-cluster-for-karpenter: ## Create test AKS cluster (with msi, cilium,
     --enable-managed-identity --enable-oidc-issuer --enable-workload-identity -o none
 	az aks get-credentials --name $(AZURE_CLUSTER_NAME) --resource-group $(AZURE_RESOURCE_GROUP) --overwrite-existing
 
+## --------------------------------------
+## AWS resources
+## --------------------------------------
+
+.PHONY: deploy-aws-cloudformation
+deploy-aws-cloudformation: ## Deploy AWS CloudFormation stack
+	curl -fsSL https://raw.githubusercontent.com/aws/karpenter-provider-aws/v"${AWS_KARPENTER_VERSION}"/website/content/en/preview/getting-started/getting-started-with-karpenter/cloudformation.yaml  > "${TEMPOUT}" \
+	&& aws cloudformation deploy \
```

Comment:
In Makefile, I prefer to execute commands line by line. no need to combine them with '&&' like we do in Dockerfile.
It would be more clear.